.woodpecker/test.yml

```diff
@@ -46,12 +46,13 @@ pipeline:
       path: *when_path
 
   lint-editorconfig:
-    image: mstruebing/editorconfig-checker
+    image: docker.io/mstruebing/editorconfig-checker
     group: test
 
   securitycheck:
     group: test
-    image: aquasec/trivy:latest
+    pull: true
```

Comment:
Is this really needed as it creates a lot of traffic?

Reply:
other option would be to add a cron job to keep the latest tagged images up to date ...

Reply:
Pull won't repull if actual latest image is already cached just check if one exists, while it is some additional traffic I don't think it's that big